src/App.vue

```diff
@@ -0,0 +1,253 @@
+<script setup>
+import { ref, onMounted } from 'vue'
+
+const origins = ref([])
+const inputText = ref('')
+const placeholderURL = ref('https://hoppscotch.io')
```

Comment:
we don't need to make this a ref. i've added more context in another comment below.

```suggestion
const placeholderURL = "https://hoppscotch.io"
```

Reply:
Resolved, will raise a commit on this PR soon.